index.html

```diff
@@ -48,6 +48,158 @@ <h2 class="text">LET'S CONNECT</h2>
+   
+
+  </p>
+  <ul class="tag">
+        <li>
+          <a href="#"><img src="./images/Tag (1).png" alt="html"></a>
+        </li>
+         <li>
+          <a href="#"><img src="./images/Tag (2).png" alt="css"></a>
+         </li>
+         <li>
+          <a href="#"><img src="./images/Tag (3).png" alt="javascript"></a>
+         </li>
+         </ul> 
+  
+       
+      <div class="buttons">
+       <button class="press">See project</button> 
+      </div>
+</section>
+<section class="cardworks-two">
+  <img src="./images/Snapshoot Portfolio (2).png" alt="" class="snapshoot">
+  <div class="left-blockone"></div>
+  
+   <h4 class="heading-two">Multi-post stories</h4> 
+  
+  
+<img src="./images/Frame 2 (1).png" alt="" class="frame">
+  <p>
+    A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+  
+  <ul class="tag">
+        <li>
+          <a href="#"><img src="./images/Tag (1).png" alt="html"></a>
+        </li>
+         <li>
+          <a href="#"><img src="./images/Tag (2).png" alt="css"></a>
+         </li>
+         <li>
+          <a href="#"><img src="./images/Tag (3).png" alt="javascript"></a>
+         </li>
+         </ul> 
+      <div class="buttons">
+       <button class="press">See project</button> 
+      </div> 
+
+</section>
+<section class="cardworks-three">
+  <img src="./images/Snapshoot Portfolio (3).png" alt="" class="snapshoot">
+  <div class="left-blocktwo"></div>
+  <h5 class="headings">Tonic</h5>
+<img src="./images/Frame 2 (1).png" alt="" class="frame">
+  <p>
+   A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+   <ul class="tag">
+        <li>
+          <a href="#"><img src="./images/Tag (1).png" alt="html"></a>
+        </li>
+         <li>
+          <a href="#"><img src="./images/Tag (2).png" alt="css"></a>
+         </li>
+         <li>
+          <a href="#"><img src="./images/Tag (3).png" alt="javascript"></a>
+         </li>
+         </ul> 
+      <div class="buttons">
+      <button class="press">See project</button>  
+      </div>
+</section>
+<section class="cardworks-four">
+  <img src="./images/Snapshoot Portfolio (4).png" alt="" class="snapshoot">
+  <div class="left-blockthree"></div>
+  <h6 class="heading-two">Multi-Post Stories</h6> 
+  <img src="./images/Frame 2 (1).png" alt="" class="frame">
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. 
+  </p>
+<ul class="tag">
+        <li>
+          <a href="#"><img src="./images/Tag (1).png" alt="html"></a>
+        </li>
+         <li>
+          <a href="#"><img src="./images/Tag (2).png" alt="css"></a>
+         </li>
+         <li>
+          <a href="#"><img src="./images/Tag (3).png" alt="javascript"></a>
+         </li>
+         </ul> 
+  
+      <div class="buttons">
+       <button class="press">See project</button> 
+      </div>
+</section>
+</section>
+<section class="about-me">
```

Comment:
- [ ] Kindly consider that this section has the same margin and padding as the one given in the Figma design to deliver the expected output check the images below 👇  on which parts to fix 
- your design 👇 
<img width="284" alt="Screen Shot 2023-04-27 at 19 00 06" src="https://user-images.githubusercontent.com/60146030/234937061-46d5dd50-8a22-46c4-b1ef-48d904107f86.png">


- Figma design 👇 
<img width="224" alt="Screen Shot 2023-04-27 at 19 02 33" src="https://user-images.githubusercontent.com/60146030/234937268-2bb1d318-9a99-4ae8-bb1e-249ee8e4ee8b.png">